docs/resources/project.md

```diff
@@ -110,7 +110,7 @@ Sets wal_level=logical for all compute endpoints in this project.
 All active endpoints will be suspended. Once enabled, logical replication cannot be disabled.
 See details: https://neon.tech/docs/introduction/logical-replication
 - `history_retention_seconds` (Number) The number of seconds to retain the point-in-time restore (PITR) backup history for this project.
-Default: 7 days, see https://neon.tech/docs/reference/glossary#point-in-time-restore.
+The default on Neon when creating a project is 1 day. See also https://neon.tech/docs/reference/glossary#point-in-time-restore.
```

Comment:
**(issue)** Please swap 7 with 1 to be consistent with the [comment](https://github.com/kislerdm/terraform-provider-neon/pull/124#discussion_r1839613099).

Reply:
Done!